holmes/plugins/toolsets/grafana/toolset_grafana_tempo.py

```
@@ -291,14 +321,245 @@
+                    fetch_full_trace(sorted_traces[i]) for i in slowest_indices
+                ],
+            }
+
+            # Return as YAML for readability
+            return StructuredToolResult(
+                status=ToolResultStatus.SUCCESS,
+                data=yaml.dump(result, default_flow_style=False, sort_keys=False),
+                params=params,
+            )
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)